cli/internal/scope/filter/filter.go

```diff
@@ -110,6 +113,7 @@ func (r *Resolver) getFilteredPackages(selectors []*TargetSelector) (*SelectedPa
 			allPackageSelectors = append(allPackageSelectors, selector)
 		}
 	}
+	r.Logger.Debug("Filtering packages", "prodPackageSelectors", prodPackageSelectors, "allPackageSelectors", allPackageSelectors)
```

Comment:
Nit, but the whole `prodPackageSelectors` controlled by `followProdDepsOnly` is dead code and will always be empty. (The behavior was ported from pnpm, but never actually hooked up)

Reply:
Yeah tbh I was wondering about that code